From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: ini backend: fix DEBUG handling

When compiled without "--enable-debug", the ini backend nevertheless
defines DEBUG and thus we get output.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 src/backends/ini/Makefile.am |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

Index: elektra-0.7.0-r1618/src/backends/ini/Makefile.am
===================================================================
--- elektra-0.7.0-r1618.orig/src/backends/ini/Makefile.am
+++ elektra-0.7.0-r1618/src/backends/ini/Makefile.am
@@ -1,7 +1,11 @@
 # $Id: Makefile.am 1618 2008-10-17 16:03:11Z registry@markus-raab.org $
 
+if DEBUG
+INI_DEBUG=-DDEBUG
+endif
+
 #testdir = $(develdocdir)/src/backends/ini/
-AM_CPPFLAGS = -I$(top_srcdir)/src/include -DDEBUG -D_POSIX_SOURCE -D_BSD_SOURCE
+AM_CPPFLAGS = -I$(top_srcdir)/src/include $(INI_DEBUG) -D_POSIX_SOURCE -D_BSD_SOURCE
 
 ini_sources = ini.c helpers.c parser.c ini.h ../../include/kdb.h ../../include/kdbbackend.h
 noinst_LIBRARIES = libelektra-ini.a
